rate_test.py

```diff
@@ -8,56 +9,94 @@
 import matplotlib.pyplot as plt
 
 # local import
-from utils import *
+#from utils import *
 
 # Will test how much difference between PUPPI MET and Predicted MET in ID.
 
-
 def main(args):
-
+    
     print("\n*****************************************\n")
 
-    # load Predicted MET (TTbar-1, SingleNeutrino-0)
-    Tarray_ML = np.load("{}TTbar_feature_array_MLMET.npy".format(args.TT))
-    print("reading", args.TT, "TTbar_feature_array_MLMET.npy")
-    Sarray_ML = np.load("{}SingleNeutrino_feature_array_MLMET.npy".format(args.SN))
-    print("reading", args.SN, "SingleNeutrino_feature_array_MLMET.npy")
-    Tarray_ML = np.concatenate((Tarray_ML[:, 0:1], 1+np.zeros((Tarray_ML.shape[0], 1))), axis=1)
-    Sarray_ML = np.concatenate((Sarray_ML[:, 0:1], np.zeros((Sarray_ML.shape[0], 1))), axis=1)
+    # load Predicted MET (TTbar-1, SingleNeutrino-0) 
+    Tarray_ML = np.load("{}TTbar_feature_array_MLMET.npy".format(args.input))
```

Comment:
use `os.path.join(args.input,'TTbar_feature_array_MLMET.npy')` and same for SingleNeutrino below.

Also, this code assumes very specific names for these numpy array failes. Are they produced with exactly those names by other code in this repo? If not, this should be made configurable.